### DISCUSSION TITLE
Add rows & columns to locked view [May 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5653] [Discord Threads]
> 🙋‍♂️ **semighoti**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Currently no.    
####    
#### We would like to understand the need for this feature (your use case). Doesn't that defeat the purpose of Locked view? Do you intend to keep existing records locked & allow users to add new content on top? You can raise a feature request in github https://github.com/nocodb/nocodb/issues/new/choose  
#  
> 🙋‍♂️ **semighoti**  
#### I just think that Locked View and Shared View basically the same atm. If we can make cells editable + add new rows (not add columns)  for manager level in Locked View, it will be great in term of management.   
#### Another use case I am thinking of is setting up a grid for my employees, then let them edit it without changing any filters or sorts. Editable  in Locked View can provide a great control for executive level in my opinion.  
#  
> 🙋‍♂️ **semighoti**  
#### Locked view editable feature is employed on Seatable, and t is very useful feature according to some Redditors.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1093902119301550150)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
